ci/database.sh

```diff
@@ -1,8 +1,5 @@
 #!/bin/bash
 APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/.."
 
-FILES="${APP_DIR}/sql/healthpro/*"
-for FILE in $FILES
-do
-  mysql -h 127.0.0.1 -u root -ppassw0rd circle_test < $FILE
-done
+APP_ENV=test bin/console doctrine:database:create --if-not-exists -n
+APP_ENV=test bin/console doctrine:migrations:migrate -n
```

Comment:
Noting that this (adding `APP_ENV=test` ahead of the commands) is all it took to address the parameter loading. I may still have some thoughts for HPO-869, but this wound up being the primary culprit.